packages/web-components/src/components/checkbox/checkbox.stories.ts

```diff
@@ -135,7 +142,7 @@ const actions = html`
 export const WithAILabel = {
   render: () => html`
     <div style="width: 400px">
-      <cds-checkbox-group legend-text="Group label">
+      <cds-checkbox-group legend-text="Group label" orientation="vertical">
```

Comment:
```suggestion
      <cds-checkbox-group legend-text="Group label">
```

I don't think this is needed here since it will be added from the property's `reflect`.